subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_spi_stm32.c

```diff
@@ -136,8 +136,8 @@ struct dma_stream {
 struct ec_host_cmd_spi_cfg {
 	SPI_TypeDef *spi;
 	const struct pinctrl_dev_config *pcfg;
-	size_t pclk_len;
 	const struct stm32_pclken *pclken;
+	size_t pclk_len;
```

Comment:
I was checking the structure definitions and initialization because of the C++ requirement mentioned in the previous comment. It is not required, but I just wanted to keep the same types(pointers) together if that makes sense.